devices.js

```diff
@@ -74,6 +74,10 @@ var devices_recommended = {
     "VIXMINI": "gl.inet-vixmini"
   },
 
+  "HPE Aruba": {
```

Comment:
I would remove the "HPE" from the vendor, as Aruba is the name prominently printed on the Box, device, marketing material (you name it)

Reply:
If I look at it closely from a user perspective, I see the point.